flycheck.el

```diff
@@ -6390,6 +6391,14 @@ See URL `http://www.jscs.info'."
                    (flycheck-sanitize-errors errors)))
   :modes (js-mode js2-mode js3-mode))
 
+(flycheck-define-checker javascript-standard
+  "A code style linter for the JavaScript Standard Style.
+
+See URL `https://github.com/feross/standard'."
+  :command ("standard" source)
```

Comment:
It exposes very little on the command line, the following is all that's available:

```
      -F  --format    Automatically format code. (using standard-format)
      -v, --verbose   Show error codes. (so you can ignore specific rules)
          --stdin     Read file text from stdin.
          --version   Show current version.
      -h, --help      Show usage information.
```